mkosi

```diff
@@ -557,6 +557,26 @@ def mount_image(args, workspace, loopdev, root_dev, home_dev, srv_dev, root_read
 
             umount(root)
 
+@complete_step("Assigning hostname")
```

Comment:
btw, we don't use Signed-Off-By on commits to our projects, that's a kernel thing